scripts/dev_populate_conferences.py

```diff
@@ -9,18 +9,20 @@
 from website import settings
 from website.app import init_app
 from website.conferences.model import Conference
-
+from datetime import datetime
 
 def main():
     init_app(set_backends=True, routes=False)
     populate_conferences()
 
-
 MEETING_DATA = {
     'spsp2014': {
         'name': 'Society for Personality and Social Psychology 2014',
         'info_url': None,
         'logo_url': None,
+        'location': 'Austin, TX',
```

Comment:
This seems silly to have this much data duplicated between the populate conferences script and this one. Can we DRY this up?